src/sentry/static/sentry/app/components/platformIconTile.tsx

```diff
@@ -78,9 +80,15 @@ const getColorStyles = ({monoTone, platform}) => {
   );
 };
 
+type Props = {
+  platform: keyof typeof PLATFORM_ICONS;
+  className?: string;
```

Comment:
I would move it to the last position of this object

Reply:
It looks like it is in the right spot to me. monoTone is also an optional prop.

Reply:
lol..I think I missed that